include/quicly/loss.h

```diff
@@ -226,14 +226,15 @@ inline uint32_t quicly_rtt_get_pto(quicly_rtt_t *rtt, uint32_t max_ack_delay, ui
 inline void quicly_loss_init(quicly_loss_t *r, const quicly_loss_conf_t *conf, uint32_t initial_rtt, const uint16_t *max_ack_delay,
                              const uint8_t *ack_delay_exponent)
 {
-    quicly_loss_t loss = {0};
-    loss.conf = conf;
-    loss.max_ack_delay = max_ack_delay;
-    loss.ack_delay_exponent = ack_delay_exponent;
-    loss.loss_time = INT64_MAX;
-    loss.alarm_at = INT64_MAX;
-    *r = loss;
-
+    *r = (quicly_loss_t){.conf = conf,
```

Comment:
This code is c99 compatible and so it should be fine as is. If not, let's see the actual error.